srcpkgs/gargoyle/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'gargoyle'
+pkgname=gargoyle
+version=2023.1
+revision=1
+build_style=cmake
+configure_args="-DWITH_TTS=$(vopt_if tts ON OFF) -DWITH_QT6=false"
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="SDL2-devel SDL2_mixer-devel qt5-devel fmt  fontconfig-devel
+	freetype-devel libjpeg-turbo-devel libpng-devel speech-dispatcher-devel
```

Comment:
indent wit ha single space inside of variables.

Reply:
Done